src/app/tab/time/time.component.html

```diff
@@ -1,111 +1,289 @@
 <div
   class="bracket bracket--left"
   *ngIf="clock.brackets.enabled && clock.brackets.left !== ''"
-  [ngClass]="{'dim':clock.brackets.dim}"
+  [ngClass]="{ dim: clock.brackets.dim }"
   [style.fontSize]="shared.getFontSize(clock.brackets.scaling, 5)"
   [style.transform]="shared.getOffset(clock.brackets.offset)"
-  [style.marginRight]="getPadding(clock.brackets.padding)">
-  {{clock.brackets.left}}
+  [style.marginRight]="getPadding(clock.brackets.padding)"
+>
+  {{ clock.brackets.left }}
 </div>
 <div class="clockContent">
   <div class="time" *ngIf="!clock.analog.enabled">
```

Comment:
The digital clock should be hidden if binary is enabled just like the analog one is doing here.

Reply:
done.